guidelines/DEV_GUIDE.md

```diff
@@ -41,9 +41,13 @@ Some useful commands:
 - `ng build <package>`: build a specific package
 - `npm run bump -- --major|minor|patch`: bump the library version
 - `npm run deploy[:major|minor|patch]`: bump the library version and deploy a new build
-- `npm run package <package>`: generate a new package
 
-For now, only the library generation is used, custom schematics might be added in the future to make the generation of other components possible as well.
+## Generating new packages
+
+We provided some custom schematics to make the creation of new packages easier. You can run these using the `ng generate`, specifying the `@acpaas-ui/schematics` or use the shorthand commands;
+
+- `npm run package <package>`: generate a new package
+- `npm run example <package>`: generate examples for an existing package
```

Comment:
the `example` script generates some basic examples folder scaffolding, but not enough to let the 'examples' script run flawless. e.g no index.ts files are generated.
Is this as designed? If so, maybe the guide should be updated with this info. If not the script should updated.

Reply:
should work, did work
the schematics tools are still very experimental (and flaky), try clearing node-modules and lockfiles and re-installing
if they turn out to not work, we should consider another strategy

Reply:
FYI
Clearing node-modules and lockfiles didn't do the trick.
The `angular.json`, `ng-package.examples.json` got updated and `demo.page.html` got created. That's it.
Shall I create a separate issue for this, so it won't block this PR?

Reply:
yes, open a new issue please since this is not a side effect of this pr